glossary.yml

```
@@ -4188,7 +4188,11 @@
     def: >
       A collection of code, data, and documentation that can be distributed and
       re-used. Also referred to in some languages as a [library](#library) or [module](#module).
-
+  ja:
+    term: パッケージ
+    def: >
+      持つ関数のまとまりとデーターと文書化が分散して、再使えるように環境に加えること出来のこと。
```

Suggestion: 配布や再利用が可能なコード、データおよびドキュメントの集合体。

---

This seems good to me. I've changed [Documentation](https://en.wikipedia.org/wiki/Documentation#Documentation_in_computer_science) to reflect the specific use in software field. Perhaps the Kanji wasn't clear for this usage.

```suggestion
      配布や再利用が可能なコード、データおよびソフトウェアドキュメンテーション集合体。
```

---

I understand that documentation means documentation, e.g., README? I 'feel' 文書 is okay, but to make an emphasis on it is a document related to code and/or data, what about:
配布や再利用が可能なコード、データおよびそれらに付随する文書をひとまとまりにしたもの。
(Heavily plagiarised wiki -  https://ja.wikipedia.org/wiki/%E3%83%A9%E3%82%A4%E3%83%96%E3%83%A9%E3%83%AA)

---

I still think ドキュメンテーション is more appropriate for this usage for example:
[https://ja.wikipedia.org/wiki/ソフトウェアドキュメンテーション](https://ja.wikipedia.org/wiki/%E3%82%BD%E3%83%95%E3%83%88%E3%82%A6%E3%82%A7%E3%82%A2%E3%83%89%E3%82%AD%E3%83%A5%E3%83%A1%E3%83%B3%E3%83%86%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3)
https://en.wikipedia.org/wiki/Software_documentation

---

Yes, I agree with @TomKellyGenetics here. ドキュメンテーション is very often used and I think everyone who comes to the Carpentries workshop do understand what this means.